pallets/pallet-bonded-coins/src/curves/lmsr.rs

```diff
@@ -11,16 +36,30 @@ use substrate_fixed::{
 use super::BondingFunction;
 use crate::{PassiveSupply, Precision, LOG_TARGET};
 
+/// A struct representing the input parameters for the LMSR model. This struct is used to convert
+/// the input parameters to the correct fixed-point type.
+///
+/// # Fields
```

Comment:
Why not documenting the field itself, and rely on cargo capabilities to generate the docs?

Reply:
Good point. Wasn't thinking of it: [2ae4290](https://github.com/KILTprotocol/kilt-node/pull/801/commits/2ae429067b29b8adcd5200855c7b3e4dcfa0abe2)